### ISSUE TITLE
word2vec-normalize

### ISSUE THREAD

**Issue by a170811** [aymericdamien/TensorFlow-Examples#264]
in the code-word2vec.py in the path : TensorFlow-Examples/examples/2_BasicModels/
line 155

why the X_embed is divided by  "tf.sqrt(tf.reduce_sum(tf.square(X_embed)))"
rather than "tf.sqrt(tf.reduce_sum(tf.square(X_embed) , 1 ) )" ?

Aren't they normalized one row by one ?
thank you.
